Packages/MIES/MIES_PackageSettings.ipf

```diff
@@ -136,7 +136,12 @@ static Function PS_ApplyStoredWindowCoordinate(variable JSONid, string win)
 
 	AssertOnAndClearRTError()
 	try
-		MoveWindow/W=$win left, top, right, bottom; AbortOnRTE
+		if(IsSubWindow(win))
+			// correct for pixel/points confusion
```

Comment:
I suspect the reason is:
"By default, panels are drawn using pixels if the screen resolution is 96 DPI but using points for higher-DPI settings. This gives backward compatibility on standard screens and reasonably-sized controls on high-resolution screens."

from `DisplayHelpTopic "Control Panel Resolution on Windows"`
from `DisplayHelpTopic "Control Panel Units"`

at 100% GUI scaling in windows `print screenresolution` results in 96 in IP.

Reply:
But why is that only relevant for exterior subwindows?